docs/docsite/rst/dev_guide/developing_collections_path.rst

```diff
@@ -0,0 +1,160 @@
+
+Make sure your collection contains exhaustive license information.
+Ansible is an open source project, so we encourage you to license it under one of open source licenses.
+If you plan to submit your collection for inclusion in the Ansible community package, your collection must satisfy the `licensing requirements <https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_requirements.html#collection-licensing-requirements>`_.
+
+If you have used the `collection_template repository <https://github.com/ansible-collections/collection_template>`_ we recommended earlier as a skeleton for your collection, it already contains the ``GNU GPL v3`` license.
+
+Follow a versioning convention
+==============================
+
+When releasing new versions of your collections, take the following recommended practices into consideration:
+
+* Follow a versioning convention. Using `SemVer <https://semver.org/>`_ is highly recommended.
+* Base your releases on `Git tags <https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases>`_.
+
+Understand and implement testing and CI
+=======================================
+
+This section is applicable to collections containing modules and plugins.
+
+For role testing, see the `Ansible Molecule <https://ansible.readthedocs.io/projects/molecule/>`_ project.
+
+Add tests
+---------
+
+Testing your collection ensures that your code works well and integrates with other components such as ``ansible-core``.
+
+Take a look at the following documents:
+
+* :ref:`Testing Ansible guide<developing_testing>`: provides general information about testing.
+* :ref:`Testing collections guide<testing_collections>`: contains collection-specific testing information.
+
+Implement continuous integration
+--------------------------------
+
+Now make sure when pull requests are created in your collection repository they are automatically tested using a CI tool such as GitHub Actions or Azure Pipelines.
+
+The `collection_template repository <https://github.com/ansible-collections/collection_template>`_ contains GitHub Actions `templates <https://github.com/ansible-collections/collection_template/tree/main/.github/workflows>`_ you can adjust and use to enable the workflows in your repository.
+
+Provide good contributor & maintainer documentation
+===================================================
+
+See the `collection_template/README.md <https://github.com/ansible-collections/collection_template/blob/main/README.md>`_ as an example.
+
+Publish your collection on distribution servers
+===============================================
+
+To distribute your collection and allow others to conveniently use it, publish your collection on one or more distribution servers.
+See the :ref:`Distributing collections guide<distributing_collections>` to learn how.
+
+Follow the Collection requirements
```

Comment:
Should this heading and the next one be subheadings under publish your collection? They are only applicable to someone submitting it to the ansible package for inclusion, right?

Reply:
@samccann i merged Collection requirements and Submit for inclusion sections into one. I think they should go separately with Publishing section as imo publishing is a good thing on its own.